README.md

```diff
@@ -1,24 +1,73 @@
-The website for Urbit.
+The website for Urbit is maintained by [Tlon](https://tlon.io) and the Urbit community. Issues and contributions are welcome.
 
 ## Getting Started
 
-Install Node 14. [nvm](https://github.com/nvm-sh/nvm) should handle this by running `nvm install 14`, then `nvm use` whenever in the repository (or automate it with a script - see [here](https://github.com/nvm-sh/nvm#deeper-shell-integration)).
+1. Fork the [Urbit site repository](https://github.com/urbit/urbit.org).
 
-Install our dependencies:
+2. [Create a local clone](https://help.github.com/articles/cloning-a-repository/) of your fork.
 
-```bash
-npm install
-```
+3. Navigate to the `urbit.org` directory and install Node 14 using [nvm](https://github.com/nvm-sh/nvm):
```

Comment:
If we’re going to enforce nvm then why are we being so novice oriented with teaching people how to use Git? GitHub’s own docs do this.